script.js

```diff
@@ -1,4 +1,159 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+/*
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+How many game modes:
+1. Player 1 click submit to roll dice.
+2. Player 1 click 1 or 2 to choose order of dice. 
+3. Player 2 click submit to roll dice. 
+4. Player 2 click 1 or 2 to choose order of dice. 
+5. Determine who gets the higher combined number.  
+*/
+
+//global variables for current game mode
+var playerRoll1 = 0;
+var playerRoll2 = 0;
+var gameMode = "start game by rolling 2 dice";
+var currentPlayer = 1;
+var allDiceOrder = [];
+var accumulativeRound = 0;
+
+var playerRollDice = function () {};
+
+// random dice generator for user to roll dice
+var randomDiceGenerator = function () {
+  playerRoll1 = Math.floor(Math.random() * 6 + 1);
+  playerRoll2 = Math.floor(Math.random() * 6 + 1);
+  console.log(`dice1 output:`, playerRoll1, "& dice2 output:", playerRoll2);
```

Comment:
generally ok but I would prob still abstract the ` Math.floor(Math.random() * 6 + 1)` part into a separate function called `rollDice` and just have `player1 = rollDice()`. two benefits: 1. this function becomes much more readable 2. if you have a bug with your dice roll generation, you only need to change the code in one place rather than two.